barretenberg/Earthfile

```diff
@@ -40,8 +40,6 @@ barretenberg-acir-tests-bb:
     RUN FLOW=prove_and_verify_ultra_honk_program ./run_acir_tests.sh
     # Fold and verify an ACIR program stack using ClientIvc
     RUN FLOW=fold_and_verify_program ./run_acir_tests.sh fold_basic
```

Comment:
The bb.js tests are now created via this Earthly file, NOT the Docker file below. So all I'm really doing here is removing the Goblin flow. The other changes are just to update the tests described here to match the more up to date ones in the Docker file.